### PR TITLE
Adding planning quality to examples

### PR DESCRIPTION
**Pull request by georgiablanco**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [ ] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [ ] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [ ] Added the corresponding license to each file and add the current year of any one that you modified.
- [ ] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_evaluate_plan_quality.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# Copyright 2019 Shadow Robot Company Ltd.
```
Comment (biofotis) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_evaluate_plan_quality.py]: 2020

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py [hunk 2]
```diff
@@ -64,9 +69,26 @@
...
+arm_to_home_plan_quality = arm_commander.evaluate_given_plan(arm_to_home_plan)
+eval_arm_home_plan_quality = arm_commander.evaluate_plan_quality(arm_to_home_plan_quality)
+
+if eval_arm_home_plan_quality == 'poor':
+    rospy.logfatal("Plan quality to the home position is poor! " +
+                   "For safety please refer to the hand and arm documentation " +
+                   "for where to start the arm " +
+                   "to ensure no unexpected movements during plan and execute.")
+    rospy.loginfo("Exiting the script... ")
+    sys.exit()
```
Comment (biofotis) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: ros exit? http://wiki.ros.org/rospy/Overview/Initialization%20and%20Shutdown

**Review by biofotis — CHANGES_REQUESTED** [review]

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_evaluate_plan_quality.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+plan = robot_commander.plan_to_joint_value_target(arm_manipulator_home_joints_goal)
+plan_quality = robot_commander.evaluate_given_plan(plan)
+eval_plan_quality = robot_commander.evaluate_plan_quality(plan_quality)
+
+if eval_plan_quality != 'good':
+    rospy.logfatal("Plan quality to the home position is poor! " +
+                   "For safety please refer to the hand and arm documentation " +
+                   "for where to start the arm " +
+                   "to ensure no unexpected movements during plan and execute.")
+    rospy.loginfo("Exiting the script... ")
```
Comment (mikramarc) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_evaluate_plan_quality.py]: maybe this is unnecessary if you have a messafe in the sys exit itself?

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py [hunk 4]
```diff
@@ -64,9 +69,26 @@
...
+arm_to_home_plan = arm_commander.plan_to_joint_value_target(arm_home_joints_goal)
+arm_to_home_plan_quality = arm_commander.evaluate_given_plan(arm_to_home_plan)
+eval_arm_home_plan_quality = arm_commander.evaluate_plan_quality(arm_to_home_plan_quality)
+
+if eval_arm_home_plan_quality != 'good':
+    rospy.logfatal("Plan quality to the home position is poor! " +
+                   "For safety please refer to the hand and arm documentation " +
+                   "for where to start the arm " +
+                   "to ensure no unexpected movements during plan and execute.")
+    rospy.loginfo("Exiting the script... ")
```
Comment (mikramarc) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: same

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py [hunk 5]
```diff
@@ -33,11 +33,12 @@
 # It is recommended to run this script in simulation first.
 
 import rospy
+import sys
 from sr_robot_commander.sr_arm_commander import SrArmCommander
 from sr_robot_commander.sr_hand_commander import SrHandCommander
 from sr_robot_commander.sr_robot_commander import SrRobotCommander
 
-rospy.init_node("right_hand_arm_joint_pos", anonymous=True)
+rospy.init_node("right_hand_arm_joint_pos", anonymous=True, disable_signals=True)
```
Comment (mikramarc) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py]: why are we disabling signals here?

**Review by mikramarc — CHANGES_REQUESTED** [review]

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py (hunk 5 above)
Comment (georgiablanco) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py]: oh yea sorry was trying something. removing now

**Review by biofotis — APPROVED** [review]

**Review by mikramarc — APPROVED** [review]

**Pull request merged**